Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/gardenlinux/gardenlinux:1592.11 AS builder
+FROM ghcr.io/gardenlinux/gardenlinux:1877.1 AS builder
```

Comment:
on `1877.1` `python3.12` is no longer available. the only available version on that image would be `python3.13`
#734 would handle this by also bumping the python version.